tasks/main.yml

```diff
@@ -63,3 +63,11 @@
   become: yes
   apt:
     deb: '{{ gitkraken_download_dir }}/{{ gitkraken_redis_filename }}'
+
+- name: symlink backup directory
+  file:
+    dest: '/home/{{ os_username }}/.gitkraken'
+    src:  '{{ gitkraken_backup_src_dir }}'
+    state: link
+  when: gitkraken_backup_use_symlink and gitkraken_backup_src_dir is defined
```

Comment:
I'm pretty sure `gitkraken_backup_src_dir is defined` will always be `true` since you defined `gitkraken_backup_src_dir` under defaults and AFAIK there's no way to undefine a variable (which is why Ansible has `omit`). You do however need to check the value is not empty-string, none or omitted i.e. `not in ('', None, omit)`.